cmd/virt-tail/main.go

```diff
@@ -136,10 +140,8 @@ func (v *VirtTail) watchFS() error {
 		return err
 	}
 
-	// initial timeout for serial console socket creation
-	const initialSocketTimeout = time.Second * 20
 	socketCheckCh := make(chan int)
-	time.AfterFunc(initialSocketTimeout, func() {
+	time.AfterFunc(*v.socketTimeout, func() {
```

Comment:
I guess there's no need for a nil check since pflag will default this?

Reply:
Yes, so we don't need to check nil here.